server/static.ts

```diff
@@ -250,7 +250,7 @@ app.get("/*", async (req, res) => {
   console.log(`Don't know how to mock: ${req.path}`, req.query);
   res
     .status(404)
-    .sendFile(path.join(STATIC_ROOT, "en-us", "_spas", "404.html"));
+    .sendFile(path.join(BUILD_OUT_ROOT, "en-us", "_spas", "404.html"));
```

Comment:
This was wrong all along. The SPAs end up in BUILD_OUT_ROOT which happens to have the same default as STATIC_ROOT...